app/controllers/api/v1/positions_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Api::V1
+  # Controller for Positions
+  class PositionsController < ApplicationController
+    before_action :set_position, only: %i[show update destroy]
+
+    # GET /positions
+    def index
+      @positions = Position.order(:id)
+
+      render json: @positions
+    end
+
+    # GET /positions/1
+    def show
+      render json: @position
+    end
+
+    # POST /positions
+    def create
+      @position = Position.new(position_params)
```

Comment:
For this POST, please provide a sample JSON body to give. Also a sample response JSON if applicable.